exporters/pelorus/utils.py

```diff
@@ -142,3 +145,20 @@ def format(self, record):
             return logging.Formatter.format(self, record)
         finally:
             self._style._fmt = prior_format
+
+
+def get_env_var(var_name: str, default_value: Optional[str] = None) -> str:
```

Comment:
OH.. I see :)

Reply:
The return type should be `Optional[str]`. That can happen if `default_value` is `None` and the variable is unset.